layers.go

```diff
@@ -98,7 +98,7 @@ func (bd *bpLayersDir) newBPLayer(name string) *bpLayer {
 	return &bpLayer{
 		layer{
 			path:       filepath.Join(bd.path, name),
-			identifier: fmt.Sprintf("%s:%s", bd.buildpack.ID, name),
+			identifier: fmt.Sprintf("%s+%s", bd.buildpack.ID, name),
```

Comment:
Ultimately, these become part of a file name and the `:` is invalid for a file name on Windows. This was the simplest solution. Other than a one-time cache bust, this detail isn't visible to the end user.

Reply:
The identifier is printed all over the place in the `analyzer`, `restorer`, and `exporter` logs. I think we should change the one place where we use the identifier to name a temporary file, instead of changing the identifier itself.

Reply:
Ah, that's true. OK it wasn't so bad to just change the file name. We had mistakenly thought the file name and identifier were more deeply entangled. I chose `__` instead of `+`. Logs will still show `:`.